scripts/build/build_node/Platform/Windows/install-windows-vsbuildtools.ps1

```diff
@@ -5,8 +5,8 @@ For complete copyright and license terms please see the LICENSE at the root of t
 SPDX-License-Identifier: Apache-2.0 OR MIT
 #>
 
-choco install visualstudio2022buildtools --version=17.4.1 --package-parameters "--config .\vs2022bt.vsconfig" -y
-choco install visualstudio2019buildtools --version=16.11.21 --package-parameters "--config .\vs2019bt.vsconfig" -y
+choco install visualstudio2022buildtools --version=117.4.1.0 --package-parameters "--config $pwd\vs2022bt.vsconfig" -y
```

Comment:
I guess the version number is correct, though the fact that it is an order of magnitude higher than VS2019 is strange: https://community.chocolatey.org/packages/visualstudio2022buildtools

Reply:
Yeah, not sure why that changed, the actual version number of the installer falls under the same scheme as 2019. I double checked and this is the only package on Chocolatey. I may eventually drop installation of VS from Chocolatey as MS has finally started to retain all LTSC versions of the bootstrapper instead of only a n+1 retention: https://learn.microsoft.com/en-us/visualstudio/releases/2022/release-history#evergreen-bootstrappers